src/nft/components/explore/Banner.tsx

```diff
@@ -165,7 +165,7 @@ const Banner = () => {
                 <CarouselCard
                   key={collection.address}
                   collection={collection}
-                  onClick={() => navigate(`/nfts/collection/${collection.address}`)}
+                  href={`/nfts/collection/${collection.address}`}
```

Comment:
need to remove `navigate` to resolve linter

Reply:
thx for the catch